dtcli/scripts/dt.py

```diff
@@ -552,8 +553,12 @@ def gencerts(**kwargs):
 def build(**kwargs):
     extension_dir_path = kwargs["extension_directory"]
     utils.require_dir_exists(extension_dir_path)
-
     target_dir_path = kwargs["target_directory"]
+
+    if extension_dir_path == target_dir_path or is_subdirectory(extension_dir_path, target_dir_path):
+        click.echo("Warning: extension_directory is the same as or contains target_directory\n"
```

Comment:
I really dislike the errors in the form "a or b is wrong", there are even memes about it. Can we be more precise?